src/high-level/lapack-bindings.lisp

```diff
@@ -0,0 +1,48 @@
+;;;; lapack-bindings.lisp
+;;;;
+;;;; Author: Cole Scott
+
+(in-package #:magicl)
+
+(macrolet ((def-all-lapack ()
```

Comment:
"Programs that write programs are happy programs."

This can be done more succinctly but that's not necessary for this PR.

Reply:
This is definitely gross. +1 on cleanup in future PR